app/scripts/models/user.js

```diff
@@ -524,6 +530,19 @@ define(function (require, exports, module) {
           }
           return exists;
         });
+    },
+
+    beginFlow: function (flowBeginTime) {
+      if (! this.has('flowId')) {
```

Comment:
Nope. But is there a way to get it into the resume token without setting it on the user model?